### DISCUSSION TITLE
Move Api directory outside of page directory.

### DISCUSSION THREAD

**Discussion by m-negahi** [vercel/next.js#38624] [Ideas]
### Describe the feature you'd like to request

Move API directory outside of page directory.

### Describe the solution you'd like

change next-app directory structure to this.
- node_modules
- pages
- apis
- styles
- .gitignore
- package.json
- README.md

### Describe alternatives you've considered

Just duplicate router config from pages for APIs.

**Comment by BaptisteGarcin** [comment]
Agreed, it's cleaner and less confusing to have API endpoints outside a pages folder that represents UI related files
